react-native-cli/index.js

```diff
@@ -240,7 +240,9 @@ function createProject(name, options) {
     version: '0.0.1',
     private: true,
     scripts: {
-      start: 'node node_modules/react-native/local-cli/cli.js start'
+      start: 'node node_modules/react-native/local-cli/cli.js start',
+      ios: 'react-native run-ios',
```

Comment:
Maybe that is because you have the cli installed globally? What @grabbou is saying makes sense